_includes/user-menu.html

```diff
@@ -1,101 +1,130 @@
-      <li><a href="{{docu}}/installation/synology.html">Synology</a></li>
-      <li><a href="{{docu}}/installation/designer.html">openHAB Designer</a></li>
-    </ul>
-  </li>
-  <li><a href="{{docu}}/configuration/nginx.html">Configuration</a>
-    <ul>
-      <li><a href="{{docu}}/configuration/nginx.html">Reverse Proxy</a></li>
-    </ul>
-  </li>
-  <li><a href="{{docu}}/administration/index.html">Administration</a>
-    <ul>
-      <li><a href="{{docu}}/administration/index.html">Overview</a></li>
-      <li><a href="{{docu}}/administration/console.html">Console</a></li>
-      <li><a href="{{docu}}/administration/logging.html">Logging</a></li>
-      <li><a href="{{docu}}/administration/bundles.html">Bundle Management</a></li>
-      <li><a href="{{docu}}/administration/runtime.html">Runtime Commands</a></li>
+      <li><a href="{{docu}}/installation/synology.html">Synology DiskStation</a></li>
+      <li><a href="{{docu}}/installation/qnap.html">QNAP NAS</a></li>
+      <hr />
+      <li><a href="{{docu}}/installation/designer.html">SmartHome Designer</a></li>
+      <li><a href="{{docu}}/installation/nginx.html">Authentication &amp; HTTPS</a></li>
+      <li><a href="https://my.openhab.org/docs">my.openHAB (external)</a></li>
     </ul>
   </li>
-  <li><a href="{{docu}}/features/index.html">Features</a>
+  <li><a href="{{docu}}/configuration/index.html">Configuration</a>
     <ul>
-      <li><a href="{{docu}}/features/index.html">Overview</a></li>
-      <li><a href="{{docu}}/features/items.html">Items</a></li>
-      <li><a href="{{docu}}/features/sitemap.html">Sitemaps</a></li>
-      <li><a href="{{docu}}/features/persistence.html">Persistence</a></li>
-      <li><a href="{{docu}}/features/multimedia.html">Multimedia</a></li>
-      <li><a href="{{docu}}/features/automation/ruledsl.html">Automation</a>
-        <ul>
-          <li><a href="{{docu}}/features/automation/ruledsl.html">Textual Rules</a></li>
-          <li><a href="{{docu}}/features/automation/ngrules.html">Next-Gen Rules</a></li>
-        </ul>
-      </li>
+      <li><a href="{{docu}}/configuration/index.html">Overview</a></li>
+      <hr />
+      <li><a href="{{docu}}/configuration/things.html">Things</a></li>
+      <li><a href="{{docu}}/configuration/items.html">Items</a></li>
+      <li><a href="{{docu}}/configuration/sitemaps.html">Sitemaps</a></li>
+      <li><a href="{{docu}}/configuration/transform.html">Transformations</a></li>
+      <li><a href="{{docu}}/configuration/persistence.html">Persistence</a></li>
+      <li><a href="{{docu}}/configuration/rules-dsl.html">Rules</a></li>
+      <li><a href="{{docu}}/configuration/services.html">Services</a></li>
+      <hr />
+      <li><a href="{{docu}}/configuration/packages.html">Initial Setup Packages</a></li>
+      <li><a href="{{docu}}/configuration/ui-based.html">Paper UI Discovery</a></li>
+      <li><a href="{{docu}}/configuration/ui-based.html">Paper UI Configuration</a></li>
```

Comment:
No it's not. This is the last remaining construction site in my current scope. This should be settled before merging. I was asking for someone to help provide a meaningful menu structure and content. I am not toooo familiar with Paper UI.
I would like to address the "new" configuration possibilities in this menu part. Paper UI provides multiple of these, including initial setup and the experimental Rule Engine. Which menu entries would you like to see? I'll add one document per menu entry. Before HABmin I'll add another `<hr />`.

Btw. "Next Gen Rules"/"Experimental Rule Engine" needs a catchy name. Do you have something in mind?